runbotics-orchestrator/src/main/java/com/runbotics/domain/ProcessInstanceLoopEvent.java

```diff
@@ -68,8 +72,9 @@ public class ProcessInstanceLoopEvent implements Serializable {
     @Column(name = "loop_id")
     private String loopId;
 
+    @Type(type = "json")
     @Column(name = "iterator_element")
-    private String iteratorElement;
+    private JsonNode iteratorElement;
```

Comment:
I wasn't aware of the change in java files. Add you db migration in this PR also

Reply:
ok